README.md

```diff
@@ -265,6 +266,35 @@ $schemaBuilder->table('user', function (Blueprint $table) {
 });
 ```
 
+### Sequence
+
+If you want a simple sequence to be used as a primary key, you can use `useSequence()` method.
+If `useSequence()` is called without providing a `$name`, a sequence with name `user_id_sequence` will be created
+with `start_with_counter` set with a random value between 1 and 1,000,000.
+
+```php
+$schemaBuilder->create('user', function (Blueprint $table) {
+    $table->integer('id')->useSequence();
+});
+```
+
+If you want to be more flexibility, you can also create, alter, and drop sequences directly as below.
+
+```php
+$schemaBuilder->create('user_items', function (Blueprint $table) {
+    $table->createSequence('sequence_name');
+    $table->integer('id')->useSequence('sequence_name');
```

Comment:
There seems to be no test case for this. Can you add this test case?

Reply:
Test added.